src/angular-cesium/components/ac-layer/ac-layer.component.ts

```diff
@@ -272,33 +276,33 @@ export class AcLayerComponent implements OnInit, OnChanges, AfterContentInit, On
 	}
 
 	ngOnInit(): void {
-    this.layerService.context = this.context;
+		this.layerService.context = this.context;
 		this.layerService.options = this.options;
 		this.layerService.show = this.show;
 		this.layerService.zIndex = this.zIndex;
 		this._drawerList.forEach((drawer, drawerName) => {
 			const initOptions = this.options ? this.options[drawerName] : undefined;
 			const drawerDataSources = drawer.init(initOptions);
 			// only entities drawers create data sources
-      // TODO: Causes Bad Performance
-      // if (drawerDataSources) {
+			if (drawerDataSources) {
```

Comment:
@tomermoshe why did you put this code in comment? how does it cause bad performance and is it ok to uncomment it?